### ISSUE TITLE
Add team maintainers

### ISSUE THREAD

**Issue by parsley42** [slack-go/slack#1]
Hi @nlopes - thanks for creating an org! Would you please grant "team maintainer" rights to Ollie and I so we can start managing the committers?

https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/giving-team-maintainer-permissions-to-an-organization-member

**Comment by nlopes** [comment]
Done.

**Comment by parsley42** [comment]
Hello again @nlopes. Thanks again for working with me to scale the slack maintainers, but it looks like there's another hiccup. :-/ I was trying to add @james-lawrence as the first obvious addition to the team (assuming he's just gotten busy with other things and may wish to contribute again later), but it turns out I can't add somebody to the team who isn't already a member of the organization. I did a little reading, and it looks like there are two possibilities:
1. Configure Ollie and I as admins for the slack repository, and then we can add outside collaborators as desired, who would not be members of the org or team - but this would work.
2. Add myself and/or Ollie as an Org. _owner_, who can then add members and manage teams completely. This is a bit of a big ask, but I had already made Ollie a co-owner of the "lnxjedi" org I created in preparation for maintaining a fork, in case I never heard back from you. (I'm glad I did, though - much happier with these results.)

I just double-checked with one of my own personal repositories - I'm not able to grant anybody else the ability to add other collaborators. So, I think the two possibilities above are the only options.
